armory/scenarios/base.py

```diff
@@ -0,0 +1,102 @@
+import abc
+import argparse
+import json
+import logging
+import os
+import time
+
+import coloredlogs
+
+import armory
+from armory import paths
+from armory.utils import config_loading
+
+
+logger = logging.getLogger(__name__)
+
+
+class Scenario(abc.ABC):
+    def __init__(self):
+        pass
+
+    def validate_config(self, config):
+        """
+        Validate the scenario config
+        """
+
+    def evaluate(self, config: dict):
+        """
+        Evaluate a config for robustness against attack.
+        """
+        self.validate_config(config)
+        results = self._evaluate(config)
+        if results is None:
+            logger.warning(f"{self._evaluate} returned None, not a dict")
```

Comment:
Is this an expected return? Or just a warning for improper evaluation files?

Reply:
I wanted a way to ensure evaluation files returned results. But I'm not tied to this.

Reply:
Probably something that we would want our tests to include, instead.